cryptoconditions/types/threshold_sha256.py

```diff
@@ -50,6 +52,79 @@ def __init__(self, threshold=None):
         self.threshold = threshold
         self.subconditions = []
 
+    def sign(self, message, private_keys):
+        """Signs a message
+
+        This method will take the currently configured values for the message
+        prefix and suffix and create a signature using the provided
+        ThresholdSha256 private keys.
+
+        Args:
+            message (str): message to be signed
+            # TODO: Pretty sure that also other keys than Ed25519 keys can be
+                    used to sign, as a ThresholdSha256Fulfillment can not only
+                    consist of them but also other fulfillments.
+            private_keys (list(SigningKey)) Ed25519 private key
```

Comment:
Practically you would almost never pass a list of signing keys unless you own them all